0000-relationship-between-protocols/0000-relationship-between-protocols.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Relationship between Protocols
+draft: 1
+---
+
+# Relationship between Protocols
+
+## Prerequisites
+This document assumes the reader is familiar with the following document:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+
+## Terminology
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
+- A **payment** is sending value from one to another but it doesn't mean settled necessarily that is explained below.
+- A **settlement** is ensuring that one has the paid value. For instance, executing a transaction on a certain ledger that the payee has an account on.
```

Comment:
I think this should emphasize that this is bilateral (rather than an end-to-end concern like the ILP packet going across)

Reply:
I tried to describe both `payment` and `settlement` according to your inputs.
What do you think?